src/test/java/ru/training/at/hw5/tests/listeners/steps/GivenSteps.java

```diff
@@ -0,0 +1,21 @@
+package ru.training.at.hw5.tests.listeners.steps;
+
+import io.cucumber.java.en.Given;
+import ru.training.at.hw5.context.TestContext;
+import ru.training.at.hw5.pages.HomePage;
+
+public class GivenSteps {
+
+    HomePage homePage = new HomePage(TestContext.getInstance().getDriver());
+
+    @Given("I open JDI GitHub site on {string}")
+    public void openJdiGitHubSite(String expectedTitle) {
```

Comment:
What is the purpose of using input parameter here

Reply:
Yes, I delete this input parameter